tests/synpress/specs/swap/SwapTests.ts

```diff
@@ -38,57 +47,167 @@ describe('SWAP functional tests', () => {
       })
 
     SwapPage.swap().confirmSwap()
-    cy.confirmMetamaskTransaction({ gasFee: 11 })
-
-    TransactionHelper.checkIfTxFromLocalStorageHaveNoError()
+    cy.confirmMetamaskTransaction({ gasFee: 10, gasLimit: 200000 })
 
     MenuBar.checkToastMessage('Swap')
 
     cy.wrap(null).then(() => {
       TransactionHelper.checkErc20TokenBalance(
         AddressesEnum.DXD_TOKEN_RINKEBY,
-        balanceBefore,
+        ercBalanceBefore,
         estimatedTransactionOutput,
-        100000000
+        false
       )
       TransactionHelper.checkSubgraphTransaction('DXD', 'WETH', estimatedTransactionOutput, TRANSACTION_VALUE)
     })
   })
-  it('Should swap dxd to weth', () => {
+  it('Should swap dxd to weth [TC-52]', () => {
     EtherscanFacade.erc20TokenBalance(AddressesEnum.WETH_TOKEN).then((response: { body: { result: string } }) => {
-      balanceBefore = parseInt(response.body.result)
-      console.log("BALANCE BEFORE TEST: ",balanceBefore)
+      ercBalanceBefore = parseInt(response.body.result)
+      console.log('BALANCE BEFORE TEST: ', ercBalanceBefore)
     })
 
     SwapPage.openTokenToSwapMenu()
       .chooseToken('dxd')
       .switchTokens()
-    SwapPage.getCurrencySelectors().last().click()
-    TokenMenu.chooseToken("weth")
+    SwapPage.getCurrencySelectors()
+      .last()
+      .click()
+    TokenMenu.chooseToken('weth')
     SwapPage.typeValueFrom(TRANSACTION_VALUE.toFixed(9).toString())
 
     SwapPage.getToInput()
       .should('not.have.value', '')
       .then((res: any) => {
-        console.log("ESTIMATED OUTPUT: ", res.val())
         estimatedTransactionOutput = parseFloat(res.val())
       })
 
     SwapPage.swap().confirmSwap()
-    cy.confirmMetamaskTransaction({ gasFee: 11 })
-
-    TransactionHelper.checkIfTxFromLocalStorageHaveNoError()
+    cy.confirmMetamaskTransaction({ gasFee: 10, gasLimit: 200000 })
 
     MenuBar.checkToastMessage('Swap')
 
     cy.wrap(null).then(() => {
       TransactionHelper.checkErc20TokenBalance(
         AddressesEnum.WETH_TOKEN,
-        balanceBefore,
+        ercBalanceBefore,
         estimatedTransactionOutput,
-        100000000
+        false
       )
       TransactionHelper.checkSubgraphTransaction('DXD', 'WETH', estimatedTransactionOutput, TRANSACTION_VALUE)
     })
   })
+  it('Should swap DAI to eth [TC-53]', () => {
```

Comment:
I think we should make a convention to write tokens in capital letters, such as DAI here so should ETH be also capital letters.